### DISCUSSION TITLE
Option to list search results in a new page

### DISCUSSION THREAD

**Discussion by SoumayaMauthoorMOJ** [squidfunk/mkdocs-material#5698] [Ideas]
Apologies if this feature request has already been raised!

Confluence search has two types of searches:
1) Simple search which is similar to the existing search and lists the results as a drop down
2) Advanced search where the results of the search are listed in a separate page. The advantage is that the user can return to the search page to review the different search results. 

mkdocs-material caches the search word and you can click on the search bar to list the results, but it's more intuitive to do this via a search "page". How about adding a button to display the search results to a new page?

**Comment by squidfunk** [comment]
Thanks for asking. Yes, this was already asked for multiple times and is likely not going to happen. We're currently rebuilding the search, and we might add this as an option in the future, but definitely not in the first batch of iterations, and I'm not sure if we actually need it. [You can already deep link search results](https://squidfunk.github.io/mkdocs-material/?q=search) by using [search sharing](https://squidfunk.github.io/mkdocs-material/setup/setting-up-site-search/#search-sharing) 😊

**Reply by squidfunk** [reply]
Other than that, Material for MkDocs is Open Source, so you can fork it and tailor it to your needs, if you really need it. We try to offer features that are as complementary as they can possibly be, to enable you to build out a custom and awesome documentation site.

Extension is always an option 😊 That's the beauty of Open Source.

**Comment by SoumayaMauthoorMOJ** [comment]
Thanks for the clarification :-)
